tssc/__main__.py

```diff
@@ -127,21 +211,18 @@ def main(argv=None):
     args = parser.parse_args(argv)
 
     # validate args
-    if not os.path.exists(args.config_file) or os.stat(args.config_file).st_size == 0:
-        print_error('specified -c/--config-file must exist and not be empty')
-        sys.exit(101)
+    for config_file in args.config:
+        if not os.path.exists(config_file) or os.stat(config_file).st_size == 0:
+            print_error('specified -c/--config-file must exist and not be empty')
```

Comment:
This message should also be updated to reference `--config` instead of `--config-file`